docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -417,6 +434,15 @@ Security Solution indices defined on the {kib} Advanced Settings page
 (*Kibana* → *Stack Management* → *Advanced Settings* →
 `securitySolution:defaultIndex`).
 
+|==============================================
+
+[[opt-fields-eql-query-threshold-new_terms-esql]]
+===== Optional fields for event correlation, query, threshold, new terms, and ES|QL rules
```

Comment:
Can this section be combined with [the previous one](https://github.com/elastic/security-docs/pull/4052/files#diff-b06e37b58b218d307080f6dfb75f551ca3a09a5568a1b901f49947a7013e4e20R375)? I'm having trouble discerning which fields are applicable/optional for the event correlation, query, threshold, and new terms rules, and which are only applicable/optional for ES|QL rules.

Reply:
Resolved in [5d69fe4](https://github.com/elastic/security-docs/pull/4052/commits/5d69fe43645fcac3a8527428d5392609b160b5bc).